_data/ballerinaopenapisupportswanlake.yml

```
@@ -0,0 +1,28 @@
+- title: Ballerina OpenAPI Support
+  url: #
+  sublinks:
+    - title: Using the Capabilities of the OpenAPI Tools
+      url: /learn/ballerina-openapi-support/#using-the-capabilities-of-the-openapi-tools
+      active: using-the-capabilities-of-the-openapi-tools
+      sublinks:
+        - title: Generating Ballerina Service from the OpenAPI Definition (Design-First Approach)
+          url: /learn/ballerina-openapi-support/#generating-ballerina-service-from-the-openapi-definition-design-first-approach
+          active: generate-ballerina-service-from-the-openapi-design-first-approach
+        - title: Exporting OpenAPI from a Ballerina Service (Code-First Approach)
```

```suggestion
        - title: Exporting an OpenAPI Contract from a Ballerina Service (Code-First Approach)
```
Shall we remove the part in brackets from the title? We cannot include that in the URL anyway.